librz/core/cmd/cmd_eval.c

```diff
@@ -124,22 +91,20 @@ RZ_API char *rz_core_theme_get(RzCore *core) {
 	return core->curtheme;
 }
 
-static bool dict2keylist(void *user, const char *key, const ut64 value) {
-	RzList *list = (RzList *)user;
-	rz_list_append(list, strdup(key));
-	return true;
+static int compare_strings(const char *s1, const char *s2, RZ_UNUSED void *user) {
+	return strcmp(s1, s2);
 }
 
 /**
  * \brief      Returns the list of the rizin themes.
  *
  * \param      core  The RzCore struct to use
- * \return     On success, an RzList pointer, otherwise NULL.
+ * \return     On success, an RzPVector pointer, otherwise NULL.
  */
-RZ_API RZ_OWN RzList /*<char *>*/ *rz_core_theme_list(RZ_NONNULL RzCore *core) {
+RZ_API RZ_OWN RzPVector /*<char *>*/ *rz_core_theme_list(RZ_NONNULL RzCore *core) {
```

Comment:
The same question about the name. It's better to remove the list part. `rz_core_themes()`? Or `rz_core_theme_vec()`?